Sources/MIDI/MIDIPlayer.swift

```diff
@@ -4,41 +4,7 @@
 
 import AVFoundation
 import Utilities
-
-extension MIDIPlayer: Collection {
-    /// This is a collection of AVMusicTracks, so we define element as such
-    public typealias Element = AVMusicTrack
-
-    /// Index by an integer
-    public typealias Index = Int
-
-    /// Start Index
-    public var startIndex: Index {
-        return 0
-    }
-
-    /// Ending index
-    public var endIndex: Index {
-        return count
-    }
-
-    /// Look up by subscript
-    public subscript(index: Index) -> Element {
-        return tracks[index]
-    }
-
-    /// Next index
-    /// - Parameter index: Current Index
-    /// - Returns: Next index
-    public func index(after index: Index) -> Index {
-        return index + 1
-    }
-
-    /// Rewind the sequence
-    public func rewind() {
-        currentPositionInBeats = 0
-    }
-}
+import MIDIKitIO
 
 /// Simple MIDI Player based on Apple's AVAudioSequencer which has limited capabilities
 public class MIDIPlayer: AVAudioSequencer {
```

Comment:
Let's get rid of this. It uses AVAudioEngine. Plus subclassing... ew!

Reply:
@wtholliday MIDISampler has had a little work done but it's one of the bigger things I want to revisit this week and rebuild

Reply:
There's no "if" ... we can't support AVAudioSequencer.